src/main/java/persistence/sql/Constant.java

```diff
@@ -0,0 +1,9 @@
+package persistence.sql;
+
+public class Constant {
+    private Constant() {
+    }
+    public static final String EMPTY = "";
+    public static final String WHITE_SPACE = " ";
+    public static final String NOT_NULL = "not null";
```

Comment:
```suggestion
    public static final String EMPTY = "";
    public static final String WHITE_SPACE = " ";
    public static final String NOT_NULL = "not null";
    
    private Constant() {
    }
```
자바 컨벤션상 상수가 생성자보다 위에 위치합니다 :)